src/cript/api/api.py

```diff
@@ -235,7 +232,7 @@ def __str__(self) -> str:
         ...     storage_token=os.getenv("CRIPT_STORAGE_TOKEN")
         ... ) as api:
         ...     print(api)
-        CRIPT API Client - Host URL: 'https://api.criptapp.org/api/v1'
+        CRIPT API Client - Host URL: 'https://api.criptapp.org'
```

Comment:
i'm curious is this change cascading to other aspects of the code (implications for me) or no ? I will just leave comments here and there to try to thouroughly understand as much as possible